content/blog/infrastructure-testing-got-better/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Infrastructure Testing Just Got Better with Pulumi"
```

Comment:
Sorry, I think I missed this while reviewing the Google Doc. I'm wondering if we can have a straightforward title since it's a guest blog post. Something like "Infrastructure Testing at Scale with Pulumi's Unit Testing Capabilities"? I feel like "Just Got Better" is very sales sounding and not very descriptive.

Reply:
I'd suggest something like "At Scale Infrastructure Testing With Pulumi"